src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,92 @@
+package calculator
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.shouldBe
+
+class CalculatorTest : DescribeSpec({
+    describe("Calculator") {
+        it("더하기를 수행한다.") {
+            val calculator = Calculator()
+
+            val result = calculator.calculate("1 + 2")
+            val result2 = calculator.calculate("1 + 2 + 3")
+
+            result shouldBe 3
+            result2 shouldBe 6
+        }
```

Comment:
> 더하기를 수행한다.

라는 작명은 덧셈이 정상적으로 작동하는지를 확인하는 것이라고 느껴집니다.
따라서 `calculator.calculate("1 + 2 + 3")` 를 한번더 검사하는것이 의미가 있는지는 의문이 듭니다.
위 연산을 테스트 하신 것은 덧셈을 테스트하는것에 추가하여 덧셈이 연속적으로, 연산자가 연속적으로 동작하는지를 확인하는 것이 아닌가요?

테스트도 단일 책임 원칙을 지켜 한가지의 역할만을 가지고 있어야 그 의미가 잘 전달될 것 같습니다.
```suggestion
        it("더하기를 수행한다.") {
            val calculator = Calculator()

            val result = calculator.calculate("1 + 2")

            result shouldBe 3
        }
```

Reply:
마찬가지로 아래 모든 테스트들이 한번에 여러가지 케이스를 테스트하는 것으로 보이는데 해당 부분도 같이 고쳐보신다면 좋을것 같습니다!